pyproject.toml

```
@@ -12,6 +12,7 @@ license = "MIT"
 keywords = []
 authors = [
   { name = "Artemis", email = "me@arty.li" },
+  { name = "Falkan", email = "falkanson@gmail.com"}
```

```suggestion
  { name = "Falkan", email = "falkanson@gmail.com"},
```
Minor nit :roll_eyes: but keeping the trailing comma here avoids merge conflicts with other people adding their names